src/model.js

```diff
@@ -394,7 +406,15 @@ var Model = TransisObject.extend(function() {
     loads.push(true);
 
     attrs = Object.assign({}, attrs);
-    model = IdMap.get(this, id) || new this;
+    if(typeof this.getSubclass === 'function') {
```

Comment:
You could provide a default implementation of `getSubclass` that returns `this.displayName`. Then you wouldn't need the `typeof` check.